src/palette.js

```diff
@@ -1472,3 +1472,5 @@ palette.SchemeType;
     palette.register(scheme);
   }
 })();
+
+export default palette;
```

Comment:
If you fix `demo.html` to source `src/palette.js` (since you’ve renamed the file) and open it in a browser, you’ll get:

```
SyntaxError: export declarations may only appear at top level of a module  palette.js:1476
ReferenceError: palette is not defined                                      demo.js:178:11
```

Removing the `export default palette;` line fixes the problem.